testdata/go/v3/memcached-operator/api/v1alpha1/memcached_types.go

```diff
@@ -29,21 +29,32 @@ type MemcachedSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// Size defines the number of Memcached instances
+	// The following markers will use OpenAPI v3 schema to validate the value
+	// More info: https://book.kubebuilder.io/reference/markers/crd-validation.html
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=3
+	// +kubebuilder:validation:ExclusiveMaximum=false
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Size int32 `json:"size,omitempty"`
 
-	// Foo is an example field of Memcached. Edit memcached_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// Port defines the port that will be used to init the container with the image
+	ContainerPort int32 `json:"containerPort,omitempty"`
```

Comment:
@NikhilSharmaWe we need on top pf the Port the maker `// +operator-sdk:csv:customresourcedefinitions:type=spec` to build the csv with all descriptions